webapp-django/crashstats/settings/base.py

```diff
@@ -629,3 +629,24 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
     'SYMBOLS_BUCKET_DEFAULT_LOCATION',
     None
 )
+
+# Config for when the models pull directly from socorro.external classes.
+# NOTE: This is overwritten, for tests in settings_test.py
+SOCORRO_IMPLMENTATIONS_CONFIG = {
```

Comment:
The reason is pragmatism. I don't want to go near that stuff. 

Configman does some black magic with changing that config. It's a mutable dict. 
I don't want to go near it. The only thing that uses it is the Bugs service. 
When I'm done with this PR, I'll attack the SuperSearchFields next. And when I'm done that I'm going to rewrite the work Selena did on the Bugs service using my new technique. And when I've done that I'm going to attack each and every other postgres or http service implementation.